basic-steps/src/main/java/org/jenkinsci/plugins/workflow/steps/TimeoutStep.java

```diff
@@ -13,7 +12,7 @@
  *
  * @author Kohsuke Kawaguchi
  */
-public class TimeoutStep extends AbstractStepImpl implements Serializable {
```

Comment:
(Though to your question of safety: it would cause load errors for running builds in the middle of `timeout` after an upgrade. But such builds could not have worked well today: the timeout was not enforced.)